cfy_manager/components/composer/composer.py

```diff
@@ -170,7 +178,7 @@ def remove(self):
     def start(self):
         logger.notice('Starting Cloudify Composer...')
         self._run_db_migrate()
-        service.restart(COMPOSER)
+        service.start(COMPOSER)
```

Comment:
Does this change have an impact on the systemd based job, given that it'll then go from restarting to doing nothing, meaning that presumably any configuration won't be reloaded?

Reply:
I think this is changed while I was testing and did not revet it back again, will check it